src/content/app/genome-browser/components/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -85,6 +89,24 @@ const GeneSummary = () => {
 
   const rowClasses = classNames(styles.row, styles.spaceAbove);
 
+  const onDownloadSuccess = (params: OnDownloadPayload) => {
+    const {
+      options: { gene, transcript }
+    } = params;
+
+    const selectedOptions = gene.genomicSequence
+      ? ['gene-genomic_sequence']
+      : [''];
+
+    Object.entries(transcript).forEach((entry) => {
```

Comment:
Are the transcript options here of shape `{[option: string]: boolean}`, by the way? Isn't it worth writing `forEach(([option, isSet]) { ... }` for better readability (compared to entry[0] and entry[1])?

Reply:
That's done